app/domain/models/ExpensesModel.scala

```diff
@@ -11,5 +10,5 @@ trait ExpensesModel
     with BaseSaveModel[Expenses]
     with BaseDeleteModel{
```

Comment:
@Bukarachka I see a lot of duplications, like BaseDeleteModel, BaseDeleteRepository etc, let's make it generic and reuse.